bootstrap_force_upgrade.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#
+# Simple script to install AppScale and tools from the master branch
+# Author: AppScale Team <support@appscale.com>
+
+exec &>>/var/log/appscale/bootstrap-upgrade.log
+
+# Let's get the  command line arguments.
+while [ $# -gt 0 ]; do
+        if [ "${1}" = "--local-ip" ]; then
+                shift
+                if [ -z "${1}" ]; then
+                        usage
+                fi
+                LOCAL_IP="${1}"
+                shift
+                continue
+        fi
+        usage
+done
+
+echo "Copying database files needed from /etc/appscale to a temporary location /root/database-temp."
+mkdir -p /root/database-temp
+cp -r /etc/appscale/. /root/database-temp
+cd /root/database-temp ; rm -r 2.8.0; rm -r certs; rm environment.yaml; rm home; rm VERSION
```

Comment:
you could do it in a subshell, like you did few lines down (cd /root/database-temp; rm ...)

Also, you can also use mktemp (something like TEMP_DIR=mktemp and then use it instead of /root/database-temp)